src/basic/virt.c

```diff
@@ -236,9 +275,13 @@ static int detect_vm_uml(void) {
         r = read_full_file("/proc/cpuinfo", &cpuinfo_contents, NULL);
         if (r < 0)
                 return r;
-        if (strstr(cpuinfo_contents, "\nvendor_id\t: User Mode Linux\n"))
+
+        if (strstr(cpuinfo_contents, "\nvendor_id\t: User Mode Linux\n")) {
+                log_debug("UML virtualization found in /proc/cpuinfo (%s)", cpuinfo_contents);
                 return VIRTUALIZATION_UML;
+        }
 
+        log_debug("No virtualization found in /proc/cpuinfo (%s)", cpuinfo_contents);
```

Comment:
Hm, it seems that one of my comments was lost. The "ditto" refers to this lost comment, not the the one above. Sorry for the confusion. I'll re-add the comment in the right place.